src/Core/Diff.ml

```diff
@@ -1,192 +1,192 @@
-(* This file has been adapted from camlp5:
```

Comment:
Apparently Windows really didn't like the line endings of these files? Very strange!

Reply:
Ouch, ya, we should revert these for sure (might fail the format check anyway)